contracts/Core/CollectionManager.sol

```diff
@@ -308,25 +296,21 @@ contract CollectionManager is Initializable, CollectionStorage, StateManager, Co
         return numActiveCollections;
     }
 
-    /// @inheritdoc ICollectionManager
-    function getUpdateRegistryEpoch() external view override returns (uint32) {
-        return updateRegistryEpoch;
-    }
-
     /// @inheritdoc ICollectionManager
     function getIdToIndexRegistryValue(uint16 id) external view override returns (uint16) {
+        require(collections[id].active, "Query for Inactive Collection");
         return idToIndexRegistry[id];
     }
 
     /// @inheritdoc ICollectionManager
-    function getActiveCollectionsHash() external view override returns (bytes32 hash) {
-        hash = keccak256(abi.encodePacked(getActiveCollections()));
+    function getIndexToIdRegistryValue(uint16 index) external view override returns (uint16) {
```

Comment:
Now that we are storing results for each id, why do we require the registries? Can't Index be replaced with the IDs now?

Reply:
hmm nope 
merkle pos aware only works on seq digits 
which ids are not, once any is deactivated
hence reveal must happen on index 
and for same we req reg